src/app/Welcome/Welcome.tsx

```diff
@@ -99,31 +124,36 @@ const Welcome: React.FunctionComponent<any> = props => {
       <Button
         href={authenticationService.httpLoginUrl()}
         component={'a'}
+        style={{backgroundColor: chart_color_blue_500.value}}
       >
         Go to the console
       </Button>
     );
   };
 
+  const defaultFilter = (
+    <filter>
+    </filter>
+  );
+
   return (
     <Page>
-      <BackgroundImage src={back} />
+      <BackgroundImage src={images} filter={defaultFilter}/>
       <LoginPage
         footerListVariants={ListVariant.inline}
         brandImgSrc={icon}
         brandImgAlt="Infinispan logo"
```

Comment:
not sure if necessary...
```suggestion
        brandImgAlt="{brandname} logo"
```